manim/mobject/coordinate_systems.py

```diff
@@ -1298,13 +1369,13 @@ def __init__(
             config["frame_y_radius"],
             1,
         ),
-        x_length=None,
-        y_length=None,
-        axis_config=None,
-        y_axis_config=None,
-        background_line_style=None,
-        faded_line_style=None,
-        faded_line_ratio=1,
+        x_length: Optional[float] = None,
+        y_length: Optional[float] = None,
+        axis_config: Optional[dict] = None,
+        y_axis_config: Optional[dict] = None,
+        background_line_style: Optional[dict] = None,
+        faded_line_style: Optional[dict] = None,
+        faded_line_ratio: Optional[int] = 1,
```

Comment:
@abhi3019's comment hasn't been resolved.

```suggestion
        faded_line_ratio: Optional[float] = 1,
```